docs/CONFIGURATION.md

```diff
@@ -658,12 +658,15 @@ An array of [glob pattern][] strings.  Only tags with a tag key matching one of
 the patterns are emitted.  In contrast to `tagpass`, which will pass an entire
 metric based on its tag, `taginclude` removes all non matching tags from the
 metric.  Any tag can be filtered including global tags and the agent `host`
-tag.
+tag. When used with processor plugin, metric is be modified before it is passed 
```

Comment:
I believe this is true of [processors](https://github.com/influxdata/telegraf/blob/master/models/running_processor.go#L90), [aggregators](https://github.com/influxdata/telegraf/blob/master/models/running_aggregator.go#L136), and [outputs](https://github.com/influxdata/telegraf/blob/master/models/running_output.go#L142). In that they all filter before processing.

I think it is worth being explicit that the filters are running after an input collects metrics but before other plugins are run.

Reply:
See also this RFC https://github.com/influxdata/telegraf/issues/8792 where I asked to modify after it has been processed by the plugin..